lua/ge/extensions/MPCoreNetwork.lua

```diff
@@ -109,7 +109,8 @@ local function isLauncherConnected()
 end
 local function login(identifiers)
 	log('M', 'login', 'Attempting login...')
-	send('N:'..jsonEncode(identifiers))
+	if not identifiers then identifiers = "" else identifiers = jsonEncode(identifiers) end
```

Comment:
I would clarify that this chunk is only necessary for guests
```suggestion
	if not identifiers then identifiers = "" else identifiers = jsonEncode(identifiers) end -- guest login fix
```